include/UIKit/UIMenuController.h

```diff
@@ -40,6 +40,8 @@ extern NSString* const UIMenuControllerMenuFrameDidChangeNotification;
 
 @class UIView, UIWindow;
 
+UIKIT_EXPORT_CLASS
```

Comment:
in comments of original issue, UIRefreshControl has been called out to miss this as well. Mind adding UIKIT_EXPORT_CLASS  too to UIRefreshControl?